docs/guides/connecting_to_your_data/database/redshift.md

```diff
@@ -43,12 +43,12 @@ This will allow you to <TechnicalTag tag="validation" text="Validate" /> and exp
 
 Import these necessary packages and modules.
 
```

Comment:
One small nit here is that the Redshift `YAML` configuration is the only one where the snippet is "within" the configuration (rather than around it). Would it be worth adjusting for it  for consistency?

Reply:
I agree - will change for consistency. Commit coming up 👍

Reply:
Done! See https://github.com/great-expectations/great_expectations/pull/7199/commits/200872db49dd5dc77cb2af7fce7d4a916ca3121b